src/misc/misc.c

```diff
@@ -161,6 +161,24 @@ void print_dims(int D, const long dims[D])
 
 
 
+void debug_print_bits(int dblevel, int D, unsigned int bitmask)
+{
+	bool dbl = debug_logging;
+	debug_logging = false;
+	debug_printf(dblevel, "[");
+
+	 unsigned int maxPow = 1;
+
+	 while (maxPow < 1u << (D-1)) {
```

Comment:
BART coding style: spaces around all operators, brackets around sub-expression, and empy line after {